src/components/lightbox/lightbox.js

```diff
@@ -0,0 +1,91 @@
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import initComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+import eventedShowHideState from '../../globals/js/mixins/evented-show-hide-state';
+import eventMatches from '../../globals/js/misc/event-matches';
+import on from '../../globals/js/misc/on';
+// import eventMatches from '../../globals/js/misc/event-matches';
+
+class Lightbox extends mixin(createComponent, initComponentBySearch) {
+  constructor(element, options) {
+    super(element, options);
+    this.activeIndex = this.element.dataset.lightboxIndex;
+    this.totalSlides =
+      [...this.element.querySelectorAll(this.options.selectorLightboxItem)].length - 1;
+
+    this.updateSlide();
+    this.updateFilmstrip();
+
+    this.element.addEventListener('click', evt => {
```

Comment:
Given `.handleClick()` method is an arrow function, we don’t need to wrap it with another arrow function.